In [1]:
import pandas as pd

In [2]:
path = "matomo-org/matomo"

In [3]:
data = pd.read_csv(f"{path}.csv")
data

number                                              title  \
0       19802     [automatic Intl data updates from CLDR 41.0.0]   
1       19801  Move public environment outside private direct...   
2       19800  Is there a way to filter a specific user ID on...   
3       19799              Custom Report totals row default true   
4       19798  Updated the text and links in the Matomo Help ...   
...       ...                                                ...   
19080    9804                       Help needed with log in page   
19081    9803           Improve security of *.piwik.org services   
19082    9802  Update documentation: Placement of JavaScript ...   
19083    9801  use consistent wording "processed" to make it ...   
19084    9800         Fix unreplaced translations in transitions   

                                             description  \
0      Updated Intl plugin data with changes from CLD...   
1      The security check feature called Required (an...   
2      For example I will search user_id = 1.\r\r\n\r...   
3      Hi,\r\r\n\r\r\nit would be nice to have a flag...   
4      ### Description:\r\r\n\r\r\nWe are updating so...   
...                                                  ...   
19080  Hi There, \r\n\r\nthis is really a silly issue...   
19081  https://securityheaders.io/?q=https%3A%2F%2Fde...   
19082  This had me quite confused - please update the...   
19083  we found this was confusing while reviewing ou...   
19084  Another case where `%s` has been replaced manu...   

                                            labels  
0                                              NaN  
1                                     Enhancement,  
2                                   Potential Bug,  
3                                     Enhancement,  
4                                    Needs Review,  
...                                            ...  
19080                                    answered,  
19081  c: Website matomo.org,c: Security,answered,  
19082                       c: Website matomo.org,  
19083                           Task,c: Usability,  
19084               not-in-changelog,Needs Review,  

[19085 rows x 4 columns]

In [4]:
data.iloc[0]["description"]

'Updated Intl plugin data with changes from CLDR 41.0.0'

In [5]:
data.sort_values('number')

number                                              title  \
7344       1     adding support for windows phone 7 version 7.5   
7069       2                          Using alternate database.   
6992       3                                          Updates ?   
6817       4  SitesManager plugins: added group managment to...   
6809       5                           Collaboration via github   
...      ...                                                ...   
4      19798  Updated the text and links in the Matomo Help ...   
3      19799              Custom Report totals row default true   
2      19800  Is there a way to filter a specific user ID on...   
1      19801  Move public environment outside private direct...   
0      19802     [automatic Intl data updates from CLDR 41.0.0]   

                                            description          labels  
7344  UserAgentClass fails to return an OS for 7.5 u...             NaN  
7069  It seems to be using Zend_Db for data base man...            Bug,  
6992  Hi,\r\nIt looks like there is no update since ...            Bug,  
6817  Just some simple additions to make the "group"...             NaN  
6809  I found a ticket dealing with migration to git...            Bug,  
...                                                 ...             ...  
4     ### Description:\r\r\n\r\r\nWe are updating so...   Needs Review,  
3     Hi,\r\r\n\r\r\nit would be nice to have a flag...    Enhancement,  
2     For example I will search user_id = 1.\r\r\n\r...  Potential Bug,  
1     The security check feature called Required (an...    Enhancement,  
0     Updated Intl plugin data with changes from CLD...             NaN  

[19085 rows x 4 columns]

In [6]:
data.duplicated(subset=['number'], keep=False)

0        False
1        False
2        False
3        False
4        False
         ...  
19080    False
19081    False
19082    False
19083    False
19084    False
Length: 19085, dtype: bool

In [7]:
data[data.duplicated(subset=['number'], keep=False)]

Empty DataFrame
Columns: [number, title, description, labels]
Index: []

не понимаю, как может быть 19085 строк на 19802 issue